docs/tutorials/edc-cut-bell-pair-benchmarking.ipynb

```diff
@@ -0,0 +1,715 @@
+    "            all_layouts.update(layouts)\n",
+    "            all_initial_layout += initial_layout\n",
+    "\n",
+    "        sorted_layouts = dict(\n",
+    "            sorted(all_layouts.items(), key=lambda item: item[1])\n",
+    "        )  # sort layouts by mse\n",
+    "\n",
+    "        # get layouts and mses\n",
+    "        layout_list = list(sorted_layouts.keys())\n",
+    "        mse_list = np.asarray(list(sorted_layouts.values()))\n",
+    "\n",
+    "        # convert to numpy\n",
+    "        x = np.array(mse_list)\n",
+    "        y = np.arange(1, len(x) + 1) / len(x)\n",
+    "\n",
+    "        # Prepend (x[0], 0) to start CDF at zero\n",
+    "        x = np.insert(x, 0, x[0])\n",
+    "        y = np.insert(y, 0, 0)\n",
+    "\n",
+    "        # Create the plot\n",
+    "        plt.plot(\n",
+    "            x,\n",
+    "            y,\n",
+    "            marker=\"x\",\n",
+    "            linestyle=\"-\",\n",
+    "            label=f\"qubits: {sorted(list(set(all_initial_layout)))}\",\n",
+    "        )\n",
+    "\n",
+    "        # add qubit labels for the edge pairs\n",
+    "        for xi, yi, q in zip(x[1:], y[1:], layout_list):\n",
+    "            plt.annotate(\n",
+    "                [q[0], q[3]],\n",
+    "                (xi, yi),\n",
+    "                textcoords=\"offset points\",\n",
+    "                xytext=(5, -10),\n",
+    "                ha=\"left\",\n",
+    "                fontsize=8,\n",
+    "            )\n",
+    "\n",
+    "    plt.xscale(\"log\")\n",
+    "    plt.xlabel(\"Mean squared error of ⟨ZZ⟩ and ⟨XX⟩\")\n",
+    "    plt.ylabel(\"Cumulative distribution function\")\n",
+    "    plt.title(\"CDF for different initial layouts\")\n",
+    "    plt.grid(alpha=0.3)\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a16f346",
+   "metadata": {},
+   "source": [
+    "## Step 1: Map classical inputs to a quantum problem\n",
+    "\n",
+    "The first step is to create a set of quantum circuits to benchmark all candidate Bell-pair links tailored to the device’s topology. We programmatically search the device coupling map for all linearly connected chains of 4 qubits. Each such chain (labelled by qubit indices $[q0–q1–q2–q3]$) will serve as a test case for the entanglement-swapping circuit. By identifying all possible length-4 paths, we ensure maximum coverage for possible grouping of qubits that could realize the protocol."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55f49481",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "service = QiskitRuntimeService()\n",
+    "backend = service.least_busy(\n",
+    "    operational=True, simulator=False, dynamic_circuits=True\n",
+    ")\n",
+    "if \"if_else\" not in backend.target.operation_names:\n",
+    "    backend.target.add_instruction(IfElseOp, name=\"if_else\")"
```

Comment:
Adding `IfElseOp` is typically not needed. A backend that supports DC will already have `if_else` in the supported instruction. 

_However,_ we should have a separate cell describing the instructions for early access, given that's the only way to run DC jobs for at least a few more weeks. We'll just have to remember to remove that later. 
So this entails to doing 

```python
backend = service.least_busy(operational=True)
backend.target.add_instruction(IfElseOp, name="if_else")
```
It shouldn't use `dynamic_circuits=True` as a filter, because none of the backends support DC by default (i.e. early access only). `simulator=False` is also not needed since we don't have cloud simulators anymore.